config/webpack.config.js

```diff
@@ -245,41 +251,94 @@ module.exports = env => {
+          if (tag.attributes.src.match(/style/)) return tags;
+
+          // Force polyfills.entry.js to be first (and set `nomodules`), since
+          // vendor.entry.js gets put first even with chunksSortMode: 'manual'.
+          return tag.attributes.src.match(/polyfills/)
+            ? [
+                { ...tag, attributes: { ...tag.attributes, nomodule: true } },
+                ...tags,
+              ]
+            : [...tags, tag];
+        }, [])
+        .join('');
+
+    const generateLandingPage = ({
+      appName,
+      entryName = 'static-pages',
+      loadingMessage = 'Please wait while we load the application for you.',
+      rootUrl,
+    }) =>
+      new HtmlPlugin({
+        chunks: ['polyfills', 'vendor', 'style', entryName],
+        filename: landingPagePath(rootUrl),
+        inject: false,
+        scriptLoading: 'defer',
+        template: 'src/platform/landing-pages/dev-template.ejs',
+        templateParameters: {
+          entryName,
+          headerFooterData, // TODO: Get this placeholder data from another file
+          inlineScripts,
+          loadingMessage,
+          modifyScriptTags,
+          modifyStyleTags,
+        },
+        title: appName ? `${appName} | VA.gov` : 'VA.gov',
+      });
+
     baseConfig.plugins = baseConfig.plugins.concat(
       getAppManifests()
-        .filter(manifest => manifest.rootUrl)
-        // Only create a new landing page if one doesn't already exist from a
-        // previous build. This is useful for using the content build page for
-        // testing.
-        .filter(manifest => !fs.existsSync(landingPagePath(manifest.rootUrl)))
-        .map(
-          manifest =>
-            new HtmlWebpackPlugin({
-              filename: landingPagePath(manifest.rootUrl),
-              template:
-                manifest.landingPageDevTemplate ||
-                'src/platform/landing-pages/dev-template.ejs',
-              // Pass data to the tempates
-              templateParameters: {
-                // Everything from the manifest file
-                ...manifest,
-                // With some defaults
-                loadingMessage:
-                  manifest.loadingMessage ||
-                  'Please wait while we load the application for you.',
-                entryName: manifest.entryName || 'static-pages',
-                // TODO: Get this placeholder data from another file
-                headerFooterData,
-              },
-              // Don't inject all the assets into all the landing pages
-              // The assets we want are referenced in the template itself
-              inject: false,
-            }),
-        ),
+        .filter(({ rootUrl }) => rootUrl)
+        .map(generateLandingPage),
+    );
+
+    baseConfig.plugins.push(
+      new CopyPlugin({
+        patterns: [
+          {
+            from: 'src/site/assets/img',
+            to: path.join(outputPath, '..', 'img'),
+          },
+        ],
+      }),
```

Comment:
This pulls the assets directly from `src/site/assets/img` so the page doesn't seem as broken. Though I wonder what to do once that stuff gets moved out of the repo (if it does).

Reply:
Are you saying that images aren't loading somehow?

Reply:
That's right. There aren't images if there's no content build.